File: exercises/beecrowd/1079/1079.go
// https://judge.beecrowd.com/pt/problems/view/1078
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Scan(&n)

	for i := 0; i < n; i++ {
		var a, b, c float64
		fmt.Scan(&a, &b, &c)

		media := (a*2 + b*3 + c*5) / 10
		fmt.Printf("%.1f\n", media)
	}
}

File: exercises/beecrowd/1079/1079.ts
// https://judge.beecrowd.com/pt/problems/view/1079
import * as fs from "fs";

function main(): void {
	const input = fs.readFileSync('/dev/stdin', 'utf8');
	const lines = input.split(/\r?\n/);

	const n = Number(lines.shift());

	for (let i = 0; i < n; i++) {
		const [a, b, c] = lines[i].split(" ").map(Number);
		const media = (a * 2 + b * 3 + c * 5) / 10;
		console.log(media.toFixed(1));
	}
}

main();
